src/data/list/count.lean

```diff
@@ -131,6 +131,9 @@ lemma le_count_iff_repeat_sublist {a : α} {l : list α} {n : ℕ} :
   by rw ← this; apply filter_sublist,
  λ h, by simpa only [count_repeat] using h.count_le a⟩
 
+@[simp] lemma one_le_count_iff_mem {a : α} {l : list α} : 1 ≤ count a l ↔ a ∈ l :=
+by { convert list.le_count_iff_repeat_sublist, simp }
```

Comment:
Is there an easier proof of this via `count_pos`?

Reply:
ahh, yes, I knew it had to be there already. i also made `count_pos` a simp-lemma - I'm fairly unsure whether we should do that for both, or not.